src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -97,3 +100,18 @@ dec-no (no ¬p′) ¬p = ¬p′ , refl
 dec-yes-irr : (p? : Dec P) → Irrelevant P → (p : P) → p? ≡ yes p
 dec-yes-irr p? irr p with dec-yes p? p
 ... | p′ , eq rewrite irr p p′ = eq
+
+
+------------------------------------------------------------------------
+-- DEPRECATED NAMES
+------------------------------------------------------------------------
+-- Please use the new names as continuing support for the old names is
+-- not guaranteed.
+
+-- Version 1.2
+
+⌊_⌋ = isYes
```

Comment:
I'd consider `⌊_⌋` as untouchable.   This operator might have been around since a decade. There are several published papers using it.